sympy/solvers/diophantine.py

```diff
@@ -27,7 +27,7 @@
 from sympy.solvers.solveset import solveset_real
 from sympy.utilities import default_sort_key, numbered_symbols
 from sympy.utilities.misc import filldedent
-
+from sympy.core.expr import Expr
```

Comment:
* notice how imports are mostly alphabetic and move this to the correct place in that list
* add a test to `test_diophantine` that shows this issue has been fixed

Reply:
i have changed these things but i am unable to figure out why Travis CI build is failing.can you please guide me further?

Reply:
From Travis' log: 

> AssertionError: File contains trailing whitespace: /home/travis/build/sympy/sympy/sympy/solvers/tests/test_diophantine.py, line 171.`